packages/client/src/components/OrderStepper/OrderStepper.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Step, StepButton, StepLabel, Stepper } from '@mui/material';
+
+import { routes } from '../../config';
+
+const STEPS = ['Podsumowanie', 'Uzupełnij dane', 'Zapłać i zamów'] as const;
+
+const OrderStepper = ({ activeStepIndex }: { activeStepIndex: number }) => {
+  const generateLabels = (label: string) => {
+    if (label === STEPS[0]) {
+      return <StepButton href={routes.shoppingCart}>{label}</StepButton>;
+    }
+    if (label === STEPS[1]) {
+      return <StepButton href={routes.shoppingCartData}>{label}</StepButton>;
+    }
+    return <StepLabel>{label}</StepLabel>;
+  };
```

Comment:
Można całą funkcję wyciągnąć przed żeby nie była redefiniowana przy każdym renderze. Od biedy można też zmienić `STEPS` na obiekt gdzie klucz to label, a wartość route i wtedy `generateLabels` będzie jednolinijkowe, ale if it works then it works, także można zostawić tak jak teraz.